Sources/NIOPosix/System.swift

```diff
@@ -124,12 +124,20 @@ private let sysWritev = sysWritev_wrapper
 #elseif !os(Windows)
 private let sysWritev: @convention(c) (Int32, UnsafePointer<iovec>?, CInt) -> CLong = writev
 #endif
-#if !os(Windows)
+#if canImport(Android)
+private let sysRecvMsg: @convention(c) (CInt, UnsafeMutablePointer<msghdr>, CInt) -> ssize_t = recvmsg
```

Comment:
[These nullability annotations were added with NDK 27](https://android.googlesource.com/platform/bionic/+/4e274fa0a985bdcc0d0e312a8caf381271875877%5E%21/#F0) this summer, so adapt these networking APIs for that.